src/funcx_common/response_errors/error_classes.py

```diff
@@ -163,6 +163,32 @@ def __init__(self, ep_uuid: t.Union[uuid.UUID, str]) -> None:
         self.uuid = ep_uuid
 
 
+class EndpointInUse(FuncxResponseError):
+    """Endpoint with specified uuid is already registered"""
+
+    code = ResponseErrorCode.RESOURCE_CONFLICT
+    http_status_code = HTTPStatusCode.RESOURCE_CONFLICT
+
+    def __init__(self, ep_uuid: t.Union[uuid.UUID, str]) -> None:
+        ep_uuid = str(ep_uuid)
+        self.error_args = [ep_uuid]
+        self.reason = f"Endpoint id {ep_uuid} is already registered to this user"
+        self.uuid = ep_uuid
```

Comment:
I'm not fully clear on our terminology, but seems to me that "InUse" is not the same thing as "registered."  As text to an end-user, I would find the phrase "registered" to be confusing in this context.  "Of course it's registered.  I registered it yesterday!"

Suggest perhaps:

> Endpoint currently in-use. (id: {ep_uuid})